docs/_layouts/docs.html

```diff
@@ -155,6 +155,11 @@
         </a>
       </li>
       <li>
+        <a href="{{ site.baseurl }}/docs/external-volumes.html">
+          Stateful Applications Using External Volumes
```

Comment:
In fact, "external persistent volumes" is the right term (in contrast to "local persistent volumes" described below).

Our json schema is a bit confusing IMO:

```json
volumes: { containerPath: "/data", persistent: {
  name: "v1",
  size: 10
}
```

is a _local persistent volume_.

```json
volumes: { containerPath: "/data", persistent: {
  name: "v1", 
  size: 10,
  providerName: "dvdi",
  options: {...}
}
```

is an _external persistent volume_. Wondering whether the latter shouldn't be
```json
volumes: { containerPath: "/data", persistent: {
  name: "v1", 
  size: 10,
  kind: "external",
  options: {
    provider: "dvdi"
    ...
  }
}
```

with `kind` defaulting to `local`.